lan867x/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "0.9.0"
+version: "1.0.0"
```

Comment:
We are now considering that the component is stable enough and we'll not likely to introduce breaking changes?

Reply:
I was putting out v1.0.0 until it is proven that multidrop works, since I was afraid adding its support will introduce breaking changes, since it didn't and it works I consider the driver to be ready.